wp-convertkit.php

```diff
@@ -283,42 +285,138 @@ private static function _get_settings_page_link($query_args = array()) {
 
 		/**
 		 * Retrieve hosted form markup form the API and format
+		 * Supports attribute `form` for V2 of the API
+		 * and `id` for V3 of the API
 		 *
 		 * @param $attributes
 		 * @return string
 		 */
 		public static function get_form_embed($attributes) {
-			$attributes = shortcode_atts(array(
-				'form' => -1,
-			), $attributes);
 
-			$form = $attributes['form'];
+			$attributes = shortcode_atts( array(
+				'form' => -1,
+			), $attributes );
 
-			$form_id = intval(($form < 0) ? self::_get_settings('default_form') : $form);
 			$form = false;
 
-			if ($form_id == 0) {
-				return "";
-			}
+			if ( isset( $attributes['id'] ) ){
+				// There is an 'id' attribute so use v3 api to retrieve.
 
-			$forms_available = self::$api->get_resources('forms');
-			foreach($forms_available as $form_available) {
-				if($form_available['id'] == $form_id) {
-					$form = $form_available;
-					break;
+				$forms_available = self::$api->get_resources( 'forms' );
+				foreach ( $forms_available as $form_available ) {
+					if ( $form_available['id'] == $attributes['id'] ) {
+						$form = $form_available;
+						break;
+					}
 				}
-			}
 
-			$url = add_query_arg( array(
+				// TODO make v3 api call and return form markup
+
+
+			} else {
```

Comment:
Maybe we could do an `elseif ( isset( $attributes['form'] ) )` here to handle the v2 shortcode, and handle the default_form in the `else`?